SchoolMembers.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+
+namespace School
+{
+    public static class SchoolMembers
```

Comment:
Why do we need a separate class for that? Can't it be just a properly named List<Human>?

Reply:
Najpierw było jako lista, poźniej przeniosłem do innej klasy traktując to jako "dane zewnetrzne..."